client/src/views/ChallengeView.vue

```diff
@@ -0,0 +1,113 @@
+<template>
+    <div id="header-container" class="text-white">
+        <h1 class="text-md-h1 mt-10 mb-12 font-weight-medium" id="title">PICK A CHALLENGE</h1>
+
+        <div class="ma-10">
+            <h2 style="margin-bottom: 1vh;" class="text-h6 font-weight-bold">Choose your challenge to complete over the 12 days of the
+                challenge</h2>
+            <h2 class="text-h6">(Don't worry if you can't hit your daily target, just do your best!)</h2>
+        </div>
+    </div>
+    <v-sheet class="d-flex flex-wrap justify-center">
+        <v-card class="mx-auto challenge-card pa-5 card-outer" v-bind:style="{ backgroundImage: 'linear-gradient(rgba(0, 0, 0, 0.6),rgba(0, 0, 0, 0.6)), url(' + challenge.img + ')' }" theme="dark" v-for="challenge in challenges">
```

Comment:
You've got checks failing for ESLint and Prettier, you need to address these before the branch can be merged.